packages/e2e-playwright/models/dropin.ts

```diff
@@ -0,0 +1,92 @@
+import { Locator, Page } from '@playwright/test';
+import { USER_TYPE_DELAY } from '../tests/utils/constants';
+import LANG from '../../lib/src/language/locales/en-US.json';
+
+const CARD_IFRAME_TITLE = LANG['creditCard.encryptedCardNumber.aria.iframeTitle'];
+const EXPIRY_DATE_IFRAME_TITLE = LANG['creditCard.encryptedExpiryDate.aria.iframeTitle'];
+const CVC_IFRAME_TITLE = LANG['creditCard.encryptedSecurityCode.aria.iframeTitle'];
+
+const CARD_IFRAME_LABEL = LANG['creditCard.encryptedCardNumber.aria.label'];
+const EXPIRY_DATE_IFRAME_LABEL = LANG['creditCard.encryptedExpiryDate.aria.label'];
+const CVC_IFRAME_LABEL = LANG['creditCard.encryptedSecurityCode.aria.label'];
+
+const INSTALLMENTS_PAYMENTS = LANG['installments.installments'];
+const REVOLVING_PAYMENT = LANG['installments.revolving'];
+
+class Dropin {
+    readonly page: Page;
+
+    readonly rootElement: Locator;
+    readonly rootElementSelector: string;
+
+    readonly pmList: Locator;
+    readonly creditCard: Locator;
+    readonly brandsHolder: Locator;
+    // readonly cardNumberLabelElement: Locator;
+    // readonly cardNumberErrorElement: Locator;
+    // readonly cardNumberInput: Locator;
+    // readonly brandingIcon: Locator;
+    //
+    // readonly expiryDateField: Locator;
+    // readonly expiryDateLabelText: Locator;
+    // readonly expiryDateInput: Locator;
+    // readonly expiryDateErrorElement: Locator;
+    //
+    // readonly cvcField: Locator;
+    // readonly cvcLabelText: Locator;
+    // readonly cvcErrorElement: Locator;
+    // readonly cvcInput: Locator;
+    //
+    // readonly installmentsPaymentLabel: Locator;
+    // readonly revolvingPaymentLabel: Locator;
+    // readonly installmentsDropdown: Locator;
+    // readonly selectorList: Locator;
```

Comment:
I assume this is related to Card? Can we delete the commented code lines?

Reply:
done